docs/framework/install/on-windows-8.md

```diff
@@ -13,22 +13,35 @@ ms.devlang: dotnet
 
 # Install the .NET Framework on Windows 8
 
-You may need the .NET Framework to run an app on Windows 8. You can use the following instructions to help you. The instructions depend on which version of the .NET Framework you need.
+The .NET Framework is required to run many applications on Windows. You can use the following instructions to install it. You may have arrived on this page after trying to run an application and seeing the following dialog on your machine.
 
-## .NET Framework 4.7.1
+![This application could not be started](./media/this-application-could-not-be-started.png)
 
-You can install the [.NET Framework 4.7.1](https://www.microsoft.com/net/download/framework) on Windows 8.
+These instructions will help you install the .NET Framework versions you need. The .NET Framework 4.7.1 is the latest version, however, it is not supported on Windows 8.
 
-The .NET Framework 4.7.1 is included with the [Windows 10 Fall Creators Update](https://www.microsoft.com/software-download/windows10).
+## .NET Framework 4.6.1 and later
```

Comment:
"and later" doesn't belong in the header

Reply:
What would you propose instead? Removing "and later" changes the meaning.

Reply:
Just .NET Framework 4.6.1; any later versions are not supported on Windows 8.